### ISSUE TITLE
Events will be not created when hitting ENTER

### ISSUE THREAD

**Issue by aspdye** [owncloud/calendar#313]
### Steps to reproduce
1. Enter a new Event
2. Hitting ENTER 
### Expected behavior

Event should be saved (Create)
### Actual behavior

Event Editor closes and the Event is not saved.
### Server configuration

**Operating system**:
Ubuntu 14.04 LTS
**Web server:**
Apache + Nginx Reserve Proxy
**Database:**
SQLite
**PHP version:**
7.0
**ownCloud version:** (see ownCloud admin page)
9.0.0 (stable)
**Calendar version:** (see ownCloud apps page)
1.0
**Updated from an older ownCloud or fresh install:**
fresh
**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no
### Client configuration

**Browser:**
Safari
**Operating system:**
OS X El Capitan
#### Web server error log

```
Insert your webserver log here
```
#### Browser log

```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by georgehrke** [comment]
@codeling Interested? :)

**Comment by codeling** [comment]
@georgehrke yes, will look into it tomorrow!

"tomorrow" turned out to be later than I thought unfortunately...

**Comment by aspdye** [comment]
I think we can close this in #441 :grin:

**Comment by georgehrke** [comment]
@aspdye I think this one will already be fixed with 1.1
